extensions/wikia/RobotsTxt/tests/WikiaRobotsTest.php

```diff
@@ -171,12 +171,6 @@ public function testDisallowedNamespaces() {
 
 		$robotsTxtMock = $this->getWikiaRobotsTxt();
 
-		$this->assertTrue( $this->isNamespaceDisallowed( $robotsTxtMock, NS_SPECIAL ) );
```

Comment:
shouldn't we add unit tests for the new `noindex` meta robots tag then?

Reply:
done